### PR TITLE
improve naming convention w.r.t wallets and accounts

### PR DESCRIPTION
**Pull request by nuel77**
In the context of polkadex apps, a "Wallet" can be of two types - from an extension or stored locally,  both are places where multiple accounts are stored. So a wallet is defined by who can access its accounts (its private keys). Accounts stored by polkdot.js for example are only available via polkadot.js extension not through talisman. Each wallet can have multiple accounts which can be uniquely identified by the public key (address)  and can also have meta data like name and a password for decoding its private key. All accounts from extension should be promptly named extensionAccounts and ones stored locally in user browser (or on a mobile app) be calling browserAccounts or localAccounts.

so as rule of thumb: 
- if it can be identified uniquely by an address then its an "account"
- if the account's private key is stored by an extension then its an "extensionAccount"
- if the account's private key is stored in browser then its a "browserAccount" or "localAccount"

### CODE REVIEW

File: packages/trade-wallet/src/index.ts
Comment (serhii-temchenko) [inline packages/trade-wallet/src/index.ts:1]: Why this file is removed? It is an entry point to the package. Is it even compiles without it?

**Review by serhii-temchenko — CHANGES_REQUESTED** [review]

File: packages/trade-wallet/src/index.ts
Comment (nuel77) [reply packages/trade-wallet/src/index.ts:1]: the package is not removed, i renamed the entire folder from "trade-wallet" to "local-wallet" git is showing it as removed. but its just moved.

Comment (serhii-temchenko) [reply packages/trade-wallet/src/index.ts:1]: oh I see, then [trade-wallet](https://github.com/Polkadex-Substrate/polkadex-ts/tree/refactor-naming/packages/trade-wallet) folder should be removed? but it is still there with `CHANGELOG.md` file in it...

Comment (nuel77) [reply packages/trade-wallet/src/index.ts:1]: removed.

Comment (serhii-temchenko) [reply packages/trade-wallet/src/index.ts:1]: Thanks!

**Review by serhii-temchenko — APPROVED** [review]

**Pull request merged**